### ISSUE TITLE
Some or all popup ads not blocked by built in blocker

### ISSUE THREAD

**Issue by Doc-4** [kiwibrowser/src.next#279]
Sometimes when browsing less quality sites, pop up ads get through despite popups being blocked in settings

Spotty, but browse to a site which gives pop up ads with blocking on, still get popups.

**Smartphone (please complete the following information):**
 - Device: OnePlus 7T Pro
 - Android Version : A10 
 - Kiwi Browser Version 93.0.4577.18

**Comment by DUOLabs333** [comment]
The built-in blocker is not meant to be perfect: if you want something much more complete, try Ublock Origin.

**Comment by gnadelwartz** [comment]
#332 second to use ublock origin

**Comment by DUOLabs333** [comment]
Should be closed as a wontfix.
